pkg/kubelet/container/runtime.go

```diff
@@ -275,6 +275,8 @@ type PodStatus struct {
 	Namespace string
 	// IP of the pod.
 	IP string
+	// All IPs assigned to this pod
```

Comment:
I would just remove IP here and only use IPs. Should be easy enough to fix up where needed.

Reply:
I am not comfortable with the size of the change requested. Let me put it at the end of the queue.

Reply:
@khenidak I don't think it would be that big of a change, IP is only inside kubelet and wouldn't have API problems.

Reply:
@khenidak https://github.com/kubernetes/kubernetes/commit/1d6141a1d905685d211066c04fdae79f4327750d feel free to steal it.

Reply:
Doesn't this need to line up with the CRI?

Reply:
No. kubelet maintains an internal type for status. This type gets converted for api.PodStatus when calling the api to patch pod status

Reply:
@khenidak I really don't think we should carry duplicate fields here in kubelet since it's all internal. Any thoughts on the commit I referenced? https://github.com/kubernetes/kubernetes/commit/1d6141a1d905685d211066c04fdae79f4327750d

Reply:
updated as requested.